test/integration/Highlight.e2e.test.js

```diff
@@ -0,0 +1,32 @@
+// <reference types="Cypress" />
+describe('Highlight', () => {
+    beforeEach(() => {
+        cy.visit('/');
+    });
+
+    it('should create a new highlight on a document', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the empty highlight layer to be present
+        cy.getByTestId('ba-Layer--highlight');
+
+        cy.document().then(document => {
+            // Record the number of annotations
+            const { length } = document.querySelectorAll('[data-testid^="ba-AnnotationTarget"]');
+
+            // Enter highlight creation mode
+            cy.getByTestId('bp-AnnotationsControls-highlightBtn').click();
+
+            // Add a highlight annotation on the document
+            cy.selectText();
+            cy.submitReply();
+
+            // Assert that one more annotation is created on the document
+            cy.get('[data-testid^="ba-AnnotationTarget"]').should('have.length', length + 1);
+        });
+
```

Comment:
Should we assert that we can click it to make it active?

Reply:
The highlight target doesn't have `is-active` classname like region target. There's no simple solution to detect a highlight target active or not. I was planing to open a separate PR for highlight selection.

Reply:
That sounds good, the `is-active` class was never applied since we didn't style it any differently, but if that would help you with the testing, that could be added